charts/stable/vaultwarden/values.yaml

```diff
@@ -0,0 +1,61 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: vaultwarden/server
+  pullPolicy: IfNotPresent
+  tag: latest
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the Vaultwarden documentation
+# https://github.com/dani-garcia/vaultwarden/blob/main/.env.template
+
+mariadb:
+  enabled: false
+
+postgresql:
+  enabled: false
+
+
+env:
+  # TZ:
+  DATA_FOLDER: "config"
+
+service:
+  port:
+    port: 80
+    name: www
+  additionalPorts:
+    port:
+      port: 3012
+      name: websocket
+
+ingress:
+  enabled: false
+  hosts:
+    - host: bw.domain.tdl
+      paths:
+        - path: /
+          pathType: Prefix
+  tls: {}
+  # Additional ingress required for websockets until https://github.com/k8s-at-home/library-charts/issues/31 is resolved
+  additionalIngresses:
+    - enabled: false
+      nameSuffix: "websocket"
+      servicePort: 3012
+      hosts:
+        - host: bw.domain.tdl
+          paths:
+            - path: /notifications/hub
+              pathType: Prefix
+      tls: {}
```

Comment:
You can use a hosttpl to refer to the main ingress hostname for the host, thats a simpler default for most people I think.

Also:
the `notifications/hub/negotiate` should ALSO go to port 80:
`Route everything else, including /notifications/hub/negotiate, to the standard Rocket server, by default at port 80`.